data/spells/scripts/support/animate_dead_rune.lua

```diff
@@ -18,6 +18,6 @@ function onCastSpell(creature, variant, isHotkey)
 	end
 
 	creature:getPosition():sendMagicEffect(CONST_ME_POFF)
-	creature:sendCancelMessage(RETURNVALUE_NOTPOSSIBLE)
+	creature:sendTextMessage(MESSAGE_STATUS_SMALL, Game.getReturnMessage(RETURNVALUE_NOTPOSSIBLE))
```

Comment:
To not use any old function from compat, and instead use the original function. (sendCancelMessage uses the sendTextMessage inside it, so its better to use the original function as the old one may be deprecated sometime?)